lib/elixir/pages/Writing Documentation.md

```diff
@@ -102,7 +105,7 @@ However keep in mind adding `@doc false` does not make the function private. The
 
   * Move the undocumented function to a module with `@moduledoc false`, like `MyApp.Hidden`, ensuring the function won't be accidentally exposed or imported. Remember you can use `@moduledoc false` to hide a whole module and still document each function with `@doc`. Tools will still ignore the module.
 
-  * Start the function name with one or more underscores, for example, `__add__/2`, and add `@doc false`. The compiler does not import functions with leading underscores and they hint to anyone reading the code of their intended private usage.
+  * Start the function name with one or two underscores, for example, `__add__/2`, and add `@doc false`. The compiler does not import functions with leading underscores and they hint to anyone reading the code of their intended private usage.
```

Comment:
@whatyouhide I thought of this as edited. but still it's clear that it will ignore anything starting with an underscore. 

how about this?

> * Name your functions with a leading underscore, for example, `_add/2` or `__add__/2`, and add `@doc false` to it. The compiler does not import functions with leading underscores and they hint to anyone reading the code of their intended private usage. The convention is to use two underscores before and two underscores after the function name.